ingest/build-configs/ncbi/rules/curate.smk

```diff
@@ -0,0 +1,167 @@
+            | ./vendored/transform-field-names \
+                --field-map {params.field_map} \
+            | augur curate normalize-strings \
+            | ./vendored/transform-strain-names \
+                --strain-regex {params.strain_regex} \
+                --backup-fields {params.strain_backup_fields} \
+            | augur curate format-dates \
+                --date-fields {params.date_fields} \
+                --expected-date-formats {params.expected_date_formats} \
+            | ./vendored/transform-genbank-location \
+            | augur curate titlecase \
+                --titlecase-fields {params.titlecase_fields} \
+                --articles {params.articles} \
+                --abbreviations {params.abbreviations} \
+            | ./vendored/transform-authors \
+                --authors-field {params.authors_field} \
+                --default-value {params.authors_default_value} \
+                --abbr-authors-field {params.abbr_authors_field} \
+            | ./vendored/apply-geolocation-rules \
+                --geolocation-rules {input.all_geolocation_rules} \
+            | ./build-configs/ncbi/bin/transform-segment \
+                --segments {params.segments} \
+            | ./build-configs/ncbi/bin/transform-host \
+                --host-map {params.host_map} \
+            | ./build-configs/ncbi/bin/transform-to-match-fauna \
+            | ./vendored/merge-user-metadata \
+                --annotations {input.annotations} \
+                --id-field {params.annotations_id} ) 2>> {log} > {output.curated_ndjson}
+        """
+
+
+rule split_curated_ndjson_by_segment:
+    """
+    Split out the full curate NDJSON by segment, then we can deduplicate
+    records by strain name within each segment
+    """
+    input:
+        curated_ndjson="ncbi/data/curated.ndjson",
+    output:
+        metadata="ncbi/data/all_metadata_{segment}.tsv",
+        sequences="ncbi/results/sequences_{segment}.fasta",
+    params:
+        id_field=config["curate"]["output_id_field"],
+        sequence_field=config["curate"]["output_sequence_field"],
+    log:
+        "ncbi/logs/{segment}/split_curated_ndjson_by_segment.txt",
+    benchmark:
+        "ncbi/benchmarks/{segment}/split_curated_ndjson_by_segment.txt"
+    shell:
+        """
+        (cat {input.curated_ndjson} \
+            | ./build-configs/ncbi/bin/filter-ndjson-by-segment \
+                --segment {wildcards.segment} \
+            | ./build-configs/ncbi/bin/dedup-by-strain \
+            | augur curate passthru \
+                --output-metadata {output.metadata} \
+                --output-fasta {output.sequences} \
+                --output-id-field {params.id_field} \
+                --output-seq-field {params.sequence_field} ) 2>> {log}
+        """
+
+
+rule subset_metadata:
+    input:
+        metadata="ncbi/data/all_metadata_{segment}.tsv",
+    output:
+        subset_metadata="ncbi/results/metadata_{segment}.tsv",
```

Comment:
Noted by @jameshadfield on [Slack](https://bedfordlab.slack.com/archives/CD84ELG0N/p1716761204528389?thread_ts=1716605172.691119&cid=CD84ELG0N)

```suggestion
        subset_metadata="ncbi/data/metadata_{segment}.tsv",
```

Reply:
Updated in https://github.com/nextstrain/avian-flu/pull/40/commits/d5db437e33f8150a751303722e75515f194acaa8